CHANGELOG.md

```diff
@@ -1,5 +1,741 @@
+- Use SoundManager in Pressability and Touchable ([ff03698f20](https://github.com/facebook/react-native/commit/ff03698f20a123ebac8584891579a21443285b35) by [@mdvacca](https://github.com/mdvacca))
+- Allow update of state to receive null objects ([c65267ca6f](https://github.com/facebook/react-native/commit/c65267ca6ffdf68e9846c0cf912d23162e671758) by [@mdvacca](https://github.com/mdvacca))
+- Tidy up license headers ([b7c14f29cf](https://github.com/facebook/react-native/commit/b7c14f29cf391ae9b35a2011dca4e631c881dfd7) by [@zertosh](https://github.com/zertosh))
+- Add Landcastle job that requires changelog for React Native diffs ([082f293c85](https://github.com/facebook/react-native/commit/082f293c8518730451f3806dfb7e74fd3022e37e) by [@TheSavior](https://github.com/TheSavior))
+- Improve return type for ScrollView getters ([db84ae8fb3](https://github.com/facebook/react-native/commit/db84ae8fb38a1e7afc0b4c9e87897bffd532e15d) by [@TheSavior](https://github.com/TheSavior))
+- Fix ref type for Native Scroll View ([db662af5b2](https://github.com/facebook/react-native/commit/db662af5b28d0ad42abd1da67c32f2c38ff04900) by [@TheSavior](https://github.com/TheSavior))
+- Slightly more aggressive FabricUIManager teardown ([fa1c2be387](https://github.com/facebook/react-native/commit/fa1c2be387a95d90d8fcdaabb88b1c4c61e0097b) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Nit change to VirtualizedSectionList callback flow type ([7a2e83dba5](https://github.com/facebook/react-native/commit/7a2e83dba5164445345c5b853436f97b11e0efb0) by [@TheSavior](https://github.com/TheSavior))
+- VirtualizedList clean up prop flow definitions and comments ([8eea187085](https://github.com/facebook/react-native/commit/8eea187085d473c36af6bd6c2b447da9edc26a0e) by [@TheSavior](https://github.com/TheSavior))
+- ScrollView: Fix flow type for StickyHeaderComponentType ([5d2de9e54e](https://github.com/facebook/react-native/commit/5d2de9e54e03373627428ba9511d9cbc616424bd) by [@TheSavior](https://github.com/TheSavior))
+- Change ScrollView event flow types to support spreading ([dd3bf0545a](https://github.com/facebook/react-native/commit/dd3bf0545abf48f5abc471049e2ce2073ade1cd9) by [@TheSavior](https://github.com/TheSavior))
+- Remove duplicate flow type for TouchableProps ([fb5276c189](https://github.com/facebook/react-native/commit/fb5276c1897982f0be96f2e7074d839ae8fffdc8) by [@TheSavior](https://github.com/TheSavior))
+- Lists: Fix invalid prop usage in RNTesterExampleList ([da057749f8](https://github.com/facebook/react-native/commit/da057749f8be59516b7dcb5cb3a7bdb67c3d990c) by [@TheSavior](https://github.com/TheSavior))
+- Enable no-useless-escape lint rule ([90977b0e00](https://github.com/facebook/react-native/commit/90977b0e00acc6b3263502017c27094392e89478) by [@motiz88](https://github.com/motiz88))
+- Fix throttling in scroll view ([4159e20146](https://github.com/facebook
```

Comment:
This one shows up as a syntax error in Sublime because of the use of brackets in the commit message implying the beginning of a link. Not sure if it is actually a problem or not but I wanted to flag.

Reply:
I suspect Sublime is trying to be helpful about there being a space between `[2/N]` and the subsequent open parenthesis? It renders as we want it to, at least on GitHub.